vaadin-date-time-picker-flow-parent/vaadin-date-time-picker-flow-integration-tests/src/test/java/com/vaadin/flow/component/datetimepicker/DateTimePickerI18nIT.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2000-2024 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.datetimepicker;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.vaadin.flow.component.datetimepicker.testbench.DateTimePickerElement;
+import com.vaadin.tests.AbstractComponentIT;
+import com.vaadin.flow.testutil.TestPath;
+
+@TestPath("vaadin-date-time-picker/i18n")
+public class DateTimePickerI18nIT extends AbstractComponentIT {
+
+    private DateTimePickerElement dateTimePicker;
+
+    @Before
+    public void init() {
+        open();
+        dateTimePicker = $(DateTimePickerElement.class).first();
+    }
+
+    @Test
+    public void setAndRemoveDateAriaLabel_assertI18nProperty() {
```

Comment:
It seems that these tests could be rewritten into unit tests, as the only thing that check is the i18n property. I'll give it a try.

Reply:
Done.